File: report-generator/src/main/java/com/imm/report/generator/feign/model/Appointment.java
package com.imm.report.generator.feign.model;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Appointment {

	private long id;

	private Date appointmentDate;

	private long prix;

	private long tax;

	private AppointmentType appointmentType;

	private Person person;

	private Client client;
}

File: default/src/app/core/im-marketing/model/appointment.ts
import {Client} from './client';
import {Person} from './person';
import {AppointmentType} from './appointment-type';

export class Appointment {
	id: number;
	appointmentDate: Date;
	prix: number;
	tax: number;
	person: Person = new Person();
	client: Client = new Client();
	appointmentType: AppointmentType = new AppointmentType();
}
